src/client.ts

```diff
@@ -330,9 +331,33 @@ export class Client {
     }, 10 * 1000, 'reconnecting streaming service');
   }
 
-  connect () {
+  async connect (connectOpts = { keepTryingOnFailure: false, retryDelay: 5000 }) {
+    try {
+      this.logger.info('streamingClient.connect was called');
+      await this._tryToConnect();
+    } catch (e) {
+      let retriable = true;
```

Comment:
Should this be `retryable`?


EDIT:
I looked it up and apparently some people do spell it like this and its not necessarily wrong.

Reply:
I was also hung up on this but saw the same thing online so let it go... it is a little weird though